lib/errors.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+module.exports = (translate, options) => {
+
+  var logger = (options || {}).logger;
+
+  if (typeof translate !== 'function') {
+    throw new Error('Errorhandler needs a translate function');
+  }
+
+  return (err, req, res) => {
+    var content = {};
+
+    if (err.code === 'SESSION_TIMEOUT') {
+      content.title = translate('errors.session.title');
+      content.message = translate('errors.session.message');
+    }
+
+    if (err.code === 'NO_COOKIES') {
+      err.status = 403;
+      content.title = translate('errors.cookies-required.title');
+      content.message = translate('errors.cookies-required.message');
+    }
+
+    content.title = content.title || translate('errors.default.title');
+    content.message = content.message || translate('errors.default.message');
```

Comment:
Yes I believe that is the security concern, simply exposing info about the system unnecessarily.

I think it's good enough the way it is, but it might be nicer to have enhanced behaviour in debug mode.